src/main/java/org/dependencytrack/persistence/ComponentQueryManager.java

```diff
@@ -785,4 +783,31 @@ public synchronized void synchronizeIntegrityMetaComponent() {
             DbUtil.close(connection);
         }
     }
+
+    /**
+     * Returns the count of records in IntegrityMetaComponent.
+     *
+     * @return the count of records
+     */
+    public int getIntegrityMetaComponentCount() {
```

Comment:
Aggregate function is provided by JDO so maybe we can use that here.
Something like this 

    public Long getIntegrityMetaComponentCount() throws Exception {
        try (final Query<IntegrityMetaComponent> query = pm.newQuery(IntegrityMetaComponent.class)) {
            query.setResult("count(this)");
            return query.executeResultUnique(Long.class);
        }
    }